### PR TITLE
Update bundled theme changelog and asset version

### PR DESCRIPTION
**Pull request by kafleg**
Update version, tested upto, changelog and CSS version

Trac ticket: https://core.trac.wordpress.org/ticket/57857

### CODE REVIEW

File: src/wp-content/themes/twentyeleven/functions.php [hunk 1]
```diff
@@ -304,7 +304,7 @@ function twentyeleven_setup() {
  */
 function twentyeleven_scripts_styles() {
 	// Theme block stylesheet.
-	wp_enqueue_style( 'twentyeleven-block-style', get_template_directory_uri() . '/blocks.css', array(), '20230122' );
+	wp_enqueue_style( 'twentyeleven-block-style', get_template_directory_uri() . '/blocks.css', array(), '20230808' );
```
Comment (sabernhardt) [inline src/wp-content/themes/twentyeleven/functions.php:307]: Most assets do not need a newer version number. Twenty Eleven only needs to update `style.css` in the header template.

File: src/wp-content/themes/twentyeleven/readme.txt [hunk 2]
```diff
@@ -47,6 +47,11 @@ Images
 
 == Changelog ==
 
+= 4.4 =
+* Released: August 08, 2023
```
Comment (sabernhardt) [inline src/wp-content/themes/twentyeleven/readme.txt:51]: To follow the pattern with March 9 and November 1, dates do not need the zero.

**Review by sabernhardt — CHANGES_REQUESTED** [review]
Twenty Nineteen, Twenty Twenty and Twenty Twenty-One also need updates to their `package.json` and `package-lock.json` files. I already had started working on a pull request, and it might be easier to finish that than editing this one.

File: src/wp-content/themes/twentyeleven/readme.txt (hunk 2 above)
Comment (kafleg) [reply src/wp-content/themes/twentyeleven/readme.txt:51]: @sabernhardt do you want me to fix that and commit again?

Comment (sabernhardt) [reply src/wp-content/themes/twentyeleven/readme.txt:51]: I opened a separate pull request, [4865](https://github.com/WordPress/wordpress-develop/pull/4865), trying to prepare it for RC1. That PR still had some further changes, too.

**Comment by carolinan** [comment]
I believe this PR should be closed.

**Comment by audrasjb** [comment]
Closing in favor of #4865

**Pull request closed without merging**